pkg/controller/storagecluster/reconcile.go

```diff
@@ -55,7 +55,9 @@ osd_memory_target_cgroup_limit_ratio = 0.5
 `
 	monCountOverrideEnvVar = "MON_COUNT_OVERRIDE"
 	// EBS represents AWS EBS provisioner for StorageClass
-	EBS StorageClassProvisionerType = "kubernetes.io/aws-ebs"
+	EBS                 StorageClassProvisionerType = "kubernetes.io/aws-ebs"
+	CleanupPolicy                                   = "cleanupPolicy.ocs.openshift.io"
```

Comment:
Rename this to `CleanupPolicyLabel`, change the value to have no uppercase letters, and insert a comment explaining what this and the following values are used for.

Reply:
@jarrpa does the ocs-op have the Rook types for cephv1?